src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const probablePartner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )
+      }
+      {isMarried
+        ? <p className="Person__partner">{`${partnerName} is my ${probablePartner}`}</p>
+        : <p className="Person__partner">I am not married</p>
```

Comment:
seems to me it's better to use the ternary operator inside the paragraph not to duplicate it